GetDKAN/dkan
# Revisit metastore_admin's update

fixes #3579

## QA Steps

1. Spin up a 2.10.3 site with `dktl init --dkan=2.10.3`
1. Switch to this branch with `composer require "getdkan/dkan":"dev-3579-fix-metastore-admin-update"
1. `drush updb -y` should not error out.
1. The old DKAN Content view should be gone
1. The original Drupal Content view should be back
1. DKAN Dataset Content view menu item should be below the main admin's "DKAN" menu
